### ISSUE TITLE
Sentence Hiding in a lesson without sound.

### ISSUE THREAD

**Issue by Origitalus** [ToranSharma/Duo-Strength#158]
Problem occurs since just now.
The lesson is https://www.duolingo.com/skill/es/Healthcare/4

![Screenshot_2021-03-11 Learn a language for free](https://user-images.githubusercontent.com/60040210/110859515-d3de7980-82bb-11eb-9414-8b5247776c88.png)

In the Addon settings -> Sentence hiding all checkboxes are checked.
Shortcut Ctrl+Alt+H doesnt work, so I cant turn off the blur.
Firefox version 86.0

**Comment by ToranSharma** [comment]
Thanks for the report, I'll look into it.

**Comment by Origitalus** [comment]
I forgot to say, I THINK that I pushed the "enable/disable text hiding"-button in the question previous to this one.

**Comment by ToranSharma** [comment]
@Origitalus Have you encountered this issue again since reporting it, or was it just that one time?

I cannot seem to be able to reproduce it. If you can get it to happen again, could you give instructions on how to reproduce the issue.

I will see if there is anything that looks like it could cause this in the code. The only thing that comes to mind is that when you change between questions, both the previous (now marked) question and the new question are in the page briefly for the transition. There is a chance that the wrong question is being selected when the text is being hidden, but there would have to be some strange timings going on.

**Comment by Origitalus** [comment]
It was just this one time. I will write here if I encounter it again.

**Comment by Origitalus** [comment]
It's back!
![image](https://user-images.githubusercontent.com/60040210/112756954-c9282200-8fe7-11eb-8d0a-fb2d5837d371.png)

The lesson before had sound and hidden text. And I used the button to unhide the text and then hide it again. Then again unhide and hide again. Then again. So now with the text hidden, I wrote my text and pressed enter and it was correct. The rest is history.

**Comment by ToranSharma** [comment]
Okay, thank you for the update and the details of the actions leading up to this. I'll try that and see if I can also get it happen

**Comment by ToranSharma** [comment]
I got it happen! Though not with the method you outlined.

I revealed the text, typed the correct answer, marked it, then clicked the button again to hide the text before clicking the next question.

Normally when the sentence is marked right or wrong, a blurred sentence is revealed. As such the sentence hiding feature is turned off when the next sentence is shown. If the next question should not be hidden, nothing happens as it is current off. The issue was that by enabling the feature when the sentence was already marked, the feature is left enabled for the next question. So when we see that is shouldn't be hidden we shouldn't do nothing as before, but make sure the feature is disabled for this sentence.

This has been fixed in 60d9cc9, and will be released as part of v2.0.4 shortly.

Thanks again for the report and sorry it took so long to get round to fixing it.
